ui/packages/form/src/CheckboxGroup.svelte

```diff
@@ -22,7 +23,10 @@
 </script>
 
 <fieldset
-	class="gr-box overflow-hidden border-solid border border-gray-200 gr-panel"
+	class="form gr-box overflow-hidden border-solid border border-gray-200 gr-panel"
+	class:!rounded-none={form_position === "mid"}
+	class:!rounded-b-none={form_position === "first"}
+	class:!rounded-t-none={form_position === "last"}
 >
```

Comment:
We should create a blocks variant for this fieldset, and the label should be wrapped in a `legend` tag.

Reply:
Actually, i think we could use `svelte:element` here. I'll create an issue to remind me. Low priority.

Reply:
#1055